src/components/Videos.vue

```diff
@@ -123,6 +124,50 @@ export default {
 			// reset and show poster after play
 			this.$el.autoplay = false
 			this.$el.load()
+		},
+
+		// Save video player's volume and mute status
+		saveVolume() {
+			let videoVolume = {
+				volume: this.$el.volume,
+				muted: this.$el.muted
+			}
+			// use localStorage for persistent storage
+			try {
+				localStorage.viewerVideoVolume = JSON.stringify(videoVolume)
+			} catch (e) {
+				console.info('Local storage is not available, cannot save volume settings')
+
+				// store volume status as data of the root component
+				this.$root.$data.videoVolume = videoVolume
+			}
+		},
+
+		// Restore video player's volume and mute status
+		restoreVolume() {
+			let videoVolume
+			try {
+				// try to load from localStorage
+				if (localStorage.viewerVideoVolume) {
+					videoVolume = JSON.parse(localStorage.viewerVideoVolume)
+				}
+			} catch (e) {
+				console.info('Local storage is not available, cannot restore volume settings')
```

Comment:
Well we cannot really say that anymore as we have the option to save it on $root ;)

Reply:
Should removing the `console.info()` be good enough?

Reply:
I guess yes, since it doesn't make any sense :)

Reply:
Done.